### DISCUSSION TITLE
Seq integration with Freshdesk ( Seq.App.Freshdesk)

### DISCUSSION THREAD

**Discussion by mpp-olano** [datalust/seq-tickets#1600] [Help]
Seq unable to create Ticket on Freshdesk. Any suggestion
![image](https://user-images.githubusercontent.com/87758886/177308053-01aa7892-8264-451c-a7f9-c23525bb0334.png)

**Comment by nblumhardt** [comment]
Hi @mpp-olano; this kind of error usually occurs when the certificate presented by the other party (i.e. the Freshdesk API) doesn't match the name in the URL.

E.g. if you're calling `https://subdomain.freshdesk.com/x` but the certificate returned has CN `api.freshdesk.com` (or some other mismatch).

This could be a proxy issue, i.e. if you're using a proxy to intercept outbound HTTPS traffic and re-encrypt it with a self-signed certificate, or, it could be that the target URL at Freshdesk is incorrect.

Here's how the Freshdesk integration constructs the target URL:

https://github.com/distancify/Seq.App.Freshdesk/blob/master/Seq.App.Freshdesk/FreshdeskClient.cs#L17

Navigating to it yourself in a browser might reveal any issues with some more detail.

Hope this helps,
Nick

**Reply by mpp-olano** [reply]
Thanks @nblumhardt , The link you shared was helpful. I entered the full FreshDesk subdomain(subdomain.freshdesk.com) instead of just the subdomain part. Kindly see new error on  content:

![image](https://user-images.githubusercontent.com/87758886/177418386-be02c849-6915-4d94-a6a8-2fbc4b93c917.png)

**Comment by nblumhardt** [comment]
My best guess is that you've selected the option to group by property; in this case subsequent events in the same group will be sent as "replies", but the app doesn't send a "responder_id", which Freshdesk (now) seems to require for replies.

Does the app work if you disable grouping?

(The code I'm looking at is here: https://github.com/distancify/Seq.App.Freshdesk/blob/master/Seq.App.Freshdesk/FreshdeskReactor.cs#L53)

**Reply by mpp-olano** [reply]
Hi @nblumhardt, Thanks for the response. Grouping is disable on the signal.


![image](https://user-images.githubusercontent.com/87758886/177427827-90ee20d3-9c69-4a9f-a277-7ef9a6e724bc.png)


And left blank on the Freshdesk App:

![image](https://user-images.githubusercontent.com/87758886/177427756-df81044a-8da1-4c7f-acec-41457f4d5ab0.png)

**Reply by nblumhardt** [reply]
Interesting - it would seem that somehow the app is still attempting to create a Reply. Do you have anything set for "ticket type" in the app settings?

If this doesn't help, sending a screenshot of the full app configuration to us via `support@` might allow us to spot something. There's a chance it will be a Freshdesk configuration issue though, too, so I'd check over ticket creation settings in your Freshdesk project, as well.

Hope this helps,
Nick

**Reply by mpp-olano** [reply]
@nblumhardt used the Mail Plus integration. This can be closed.

**Reply by nblumhardt** [reply]
Thanks @mpp-olano
